package.json

```diff
@@ -17,5 +17,11 @@
     "engines": {
         "node": ">=0.1.90"
     },
-    "main": "./lib/index"
+    "main": "lib",
+    "files": [
+        "examples",
+        "lib",
+        "safe.js",
+        "themes"
+    ]
```

Comment:
ReadMe gets auto-included, as does package.json

LICENSE is debatable. If you want it in, ok, but the `license` key in package.json covers it.

`.npmignore` is basically the inverse of `files`, and requires an additional file. For this reason, `files` is cleaner.